app/models/library_location.rb

```diff
@@ -26,7 +26,7 @@ def folio_location_code
 
   class << self
     def library_name_by_code(code)
-      all_libraries[code]&.label
+      Folio::Types.libraries.find_by(code:)&.name(fallback_value: nil) || all_libraries[code]&.label
```

Comment:
Can `all_libraries[code]&.label` be the fallback value here?

Reply:
No? `Folio::Types.libraries.find_by(code:)` could be nil.